requirements/doc.txt

```diff
@@ -22,11 +22,11 @@ billiard==3.6.4.0
     # via
     #   -r requirements/base.txt
     #   celery
-bleach==4.1.0
+bleach==5.0.0
```

Comment:
`bleach==5.0.0` contains breaking changes related to `clean()` function used to sanitize css. Since tests are passing now, the version can be bumped.